lua/configs/alpha.lua

```diff
@@ -7,17 +7,33 @@ alpha.setup(astronvim.user_plugin_opts("plugins.alpha", {
     {
       type = "text",
       val = astronvim.user_plugin_opts("header", {
-        " █████  ███████ ████████ ██████   ██████",
-        "██   ██ ██         ██    ██   ██ ██    ██",
-        "███████ ███████    ██    ██████  ██    ██",
-        "██   ██      ██    ██    ██   ██ ██    ██",
-        "██   ██ ███████    ██    ██   ██  ██████",
-        " ",
-        "    ███    ██ ██    ██ ██ ███    ███",
-        "    ████   ██ ██    ██ ██ ████  ████",
-        "    ██ ██  ██ ██    ██ ██ ██ ████ ██",
-        "    ██  ██ ██  ██  ██  ██ ██  ██  ██",
-        "    ██   ████   ████   ██ ██      ██",
+        -- " █████  ███████ ████████ ██████   ██████",
+        -- "██   ██ ██         ██    ██   ██ ██    ██",
+        -- "███████ ███████    ██    ██████  ██    ██",
+        -- "██   ██      ██    ██    ██   ██ ██    ██",
+        -- "██   ██ ███████    ██    ██   ██  ██████",
+        -- " ",
+        -- "    ███    ██ ██    ██ ██ ███    ███",
+        -- "    ████   ██ ██    ██ ██ ████  ████",
+        -- "    ██ ██  ██ ██    ██ ██ ██ ████ ██",
+        -- "    ██  ██ ██  ██  ██  ██ ██  ██  ██",
+        -- "    ██   ████   ████   ██ ██      ██",
+        -- Logo created from Cantarell Extra Bold (LO, gIMP)
+        -- ASCII art by https://lachlanarthur.github.io/Braille-ASCII-Art/
+        "⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣀⣀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢀⣀⣀⠀⠀⠀⣀⣀⡀⠀⠀⠀⢰⣿⣦⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀",
+        "⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣿⣿⣿⠃⠀⠀⠀⠀⠀⣰⣶⡶⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢸⣿⣿⣧⠀⠀⢹⣿⡇⠀⠀⠀⠀⢻⣿⣧⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀",
+        "⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⣠⣾⣿⣿⣿⡟⠀⢀⣀⣀⣀⣠⣿⣿⣃⣀⣀⡀⠀⣀⠀⠀⢀⣀⡀⠀⢸⣿⣿⣿⣧⠀⠸⣿⣿⣀⡀⠀⠀⣀⣉⣁⣀⢀⣀⡀⠀⣀⡀⠀⠀⣀⡀⠀⠀⠀⠀⠀⠀⠀⠀⠀",
+        "⠀⠀⠀⠀⠀⠀⠀⠀⣠⣾⣿⠏⣼⣿⣿⣵⣾⣿⣿⣿⣷⣿⣿⣿⣿⣿⣿⣷⣿⣿⣧⣾⣿⣿⣿⣷⣸⣿⣿⢿⣿⣧⠀⣿⣧⢻⣷⡀⠀⣿⣿⣿⣿⡌⢿⣿⣾⣿⣿⣷⣼⣿⣿⣷⣤⡀⠀⠀⠀⠀⠀⠀",
+        "⠀⠀⠀⠀⠀⠀⣠⣾⣿⠟⠁⣼⣿⣿⣿⣿⣟⣉⠉⠛⣿⣿⡿⠛⢻⣿⣿⡿⠿⣿⣿⣿⠟⠛⣿⣿⣿⣿⣿⠈⢿⣿⣷⣸⣿⣿⣿⣷⡀⢿⣿⡿⣿⣿⡌⢿⣿⣿⠛⢿⣿⣿⡟⠻⣿⣿⣦⠀⠀⠀⠀⠀",
+        "⠀⠀⠀⠀⢠⣾⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣿⣰⣿⣿⠃⠀⣾⣿⡿⠀⢀⣿⣿⣿⠀⠀⣿⣿⣿⣿⣿⡄⠈⢻⣿⣿⣿⣿⣿⣿⣷⣸⣿⣧⢻⣿⣿⡌⣿⣿⣧⠀⠹⣿⣿⣆⠈⢻⣿⣷⡄⠀⠀⠀",
+        "⠀⠀⢀⣴⣿⡿⠿⠿⠿⢿⣿⣿⢿⣃⣈⣩⣿⣿⣿⣿⣿⣏⣀⣸⣿⣿⡇⠀⠸⣿⣿⣿⣄⣤⣿⣿⣿⣿⣿⡇⠀⠀⠻⣿⣿⣿⣷⠹⣿⣿⣿⣿⠀⢿⣿⣿⡘⣿⣿⣧⡀⠙⣿⣿⣦⠀⠙⣿⣿⣆⠀⠀",
+        "⢀⣴⣿⣿⠏⠀⠀⠀⠀⣾⣿⠁⣸⣿⣿⣿⡿⠟⠸⣿⣿⣿⣿⣿⣿⣿⠀⠀⠀⠻⣿⣿⣿⣿⣿⠟⢹⣿⣿⡇⠀⠀⠀⠹⣿⣿⣿⡆⠹⣿⣿⣿⠀⠘⣿⣿⣷⡘⣿⣿⣷⡀⠘⢿⣿⣷⡀⠈⢿⣿⣷⡄",
+        --
+        -- "⠀⠀⠀⠀⠀⣰⣿⣿⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀⢠⣤⣤⡄⠀⠀⢠⣤⡤⠀⠀⠀⠀⠀⠀⠀⠀⣰⣾⡷⠀⠀⠀⠀⠀⠀⠀⠀⠀⠀",
+        -- "⠀⠀⠀⢀⣼⡿⣿⣿⡆⠀⠀⠀⢀⣀⣀⡀⠀⣀⣿⣿⣁⡀⢀⣀⡀⢀⣀⠀⠀⠀⣀⣀⡀⠀⠀⠀⣼⣿⣿⣷⠀⠀⣸⣿⡇⣀⣀⡀⠀⠀⣀⣀⠀⣙⣋⠁⢀⣀⠀⢀⣀⡀⠀⢀⣀⡀",
+        -- "⠀⠀⢠⣾⡿⠁⣿⣿⡇⠀⢠⣾⡿⠿⠿⠏⠸⣿⣿⡿⠿⠀⣼⣿⣷⣿⡿⢀⣴⣿⠿⢿⣿⣷⠀⢰⣿⡟⢻⣿⣇⢠⣿⣿⠀⣿⣿⡇⢀⣼⣿⠋⢸⣿⡏⠀⣼⣿⡿⢿⣿⣿⡾⢿⣿⣿",
+        -- "⠀⢠⣿⣿⣷⣶⣿⣿⡇⠀⠸⢿⣿⣷⣶⡄⢀⣿⣿⠁⠀⢠⣿⣿⠁⠀⠀⣾⣿⡇⠀⢸⣿⡟⠀⣾⣿⠃⠈⣿⣿⣾⣿⠇⠀⢸⣿⣇⣾⡿⠁⠀⣿⣿⠃⢠⣿⣿⠀⢠⣿⣿⠀⢠⣿⣿",
+        -- "⣰⣿⡿⠉⠉⠉⢹⣿⣿⢠⣶⣶⣶⣿⠟⠀⠸⣿⣿⣶⠀⣼⣿⡇⠀⠀⠀⢿⣿⣷⣶⣿⠟⠁⢰⣿⡟⠀⠀⠸⣿⣿⡿⠀⠀⠸⣿⣿⡟⠀⠀⣸⣿⡟⠀⣾⣿⡇⠀⣾⣿⡇⠀⣼⣿⡇",
```

Comment:
I think I like this one the best, could you update the commit to use this one and remove the last one. @kabinspace what do you think of this updated ASCII art?

Reply:
I chatted with @kabinspace and we like how bright and contrasted the current look is. It's a bit lighter which makes it harder to read. Could you try making the font a bit more bold? If it's not as contrasted we might stick with the current Ascii art and leave it a bit simpler. Thanks for investigating this though!